### PR TITLE
feat(a380x): efis vor needle and glareshield buttons

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #[issue_no]

## Summary of Changes

- Hooks up the EFIS VOR needle buttons to toggle the VOR needles on the ND on/off.
- Hooks up the master warning and master caution buttons and indicators.
- Hooks up the autoland indicators.
- Hooks up the ATC MSG buttons and indicators.

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml [hunk 1]
```diff
@@ -8,24 +8,35 @@
...
         </UseTemplate>
 
         <!-- MASTER WARN -->
-        <UseTemplate Name="FBW_Push_Toggle">
+        <UseTemplate Name="FBW_Push_Held">
             <NODE_ID>PUSH_GLARESHIELD_CS_MASTERWARN</NODE_ID>
-            <TOOLTIPID>ACKNOWLEDGE MASTER WARNING (INOP.)</TOOLTIPID>
+            <TOOLTIPID>ACKNOWLEDGE MASTER WARNING</TOOLTIPID>
+            <HOLD_SIMVAR>L:PUSH_AUTOPILOT_MASTERAWARN_L</HOLD_SIMVAR>
+            <SEQ1_CODE>(L:A32NX_MASTER_WARNING)</SEQ1_CODE>
```
Comment (flogross89) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml:29]:
```suggestion
            <SEQ1_CODE>(L:A32NX_MASTER_WARNING) (L:A32NX_OVHD_INTLT_ANN) 0 == (L:A32NX_ELEC_DC_ESS_BUS_IS_POWERED, Bool) and or</SEQ1_CODE>
```

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml [hunk 2]
```diff
@@ -8,24 +8,35 @@
...
+            <DISABLE_SEQ2/>
```
Comment (flogross89) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml:30]: Would we also want to add SEQ1_POWERED? 
`<SEQ1_POWERED>(L:A32NX_ELEC_DC_ESS_BUS_IS_POWERED)</SEQ1_POWERED>`

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml [hunk 3]
```diff
@@ -8,24 +8,35 @@
...
         </UseTemplate>
 
         <!-- MASTER CAUT -->
-        <UseTemplate Name="FBW_Push_Toggle">
+        <UseTemplate Name="FBW_Push_Held">
             <NODE_ID>PUSH_GLARESHIELD_CS_MASTERCAUT</NODE_ID>
-            <TOOLTIPID>ACKNOWLEDGE MASTER CAUTION (INOP.)</TOOLTIPID>
+            <TOOLTIPID>ACKNOWLEDGE MASTER CAUTION</TOOLTIPID>
+            <HOLD_SIMVAR>L:PUSH_AUTOPILOT_MASTERCAUT_L</HOLD_SIMVAR>
+            <SEQ1_CODE>(L:A32NX_MASTER_CAUTION)</SEQ1_CODE>
```
Comment (flogross89) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml:38]:
```suggestion
            <SEQ1_CODE>(L:A32NX_MASTER_CAUTION) (L:A32NX_OVHD_INTLT_ANN) 0 == (L:A32NX_ELEC_DC_ESS_BUS_IS_POWERED, Bool) and or</SEQ1_CODE>
```

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml [hunk 4]
```diff
@@ -8,24 +8,35 @@
...
+            <DISABLE_SEQ2/>
```
Comment (flogross89) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml:39]: Same here, add `<SEQ1_POWERED>(L:A32NX_ELEC_DC_ESS_BUS_IS_POWERED)</SEQ1_POWERED>` ?

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml [hunk 5]
```diff
@@ -39,24 +50,35 @@
...
         </UseTemplate>
 
         <!-- MASTER WARN -->
-        <UseTemplate Name="FBW_Push_Toggle">
+        <UseTemplate Name="FBW_Push_Held">
             <NODE_ID>PUSH_GLARESHIELD_FO_MASTERWARN</NODE_ID> <!-- this typo is intentional, it's like that in the gltf -->
-            <TOOLTIPID>ACKNOWLEDGE MASTER WARNING (INOP.)</TOOLTIPID>
+            <TOOLTIPID>ACKNOWLEDGE MASTER WARNING</TOOLTIPID>
+            <HOLD_SIMVAR>L:PUSH_AUTOPILOT_MASTERAWARN_R</HOLD_SIMVAR>
+            <SEQ1_CODE>(L:A32NX_MASTER_WARNING)</SEQ1_CODE>
```
Comment (flogross89) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml:71]: See comments for CAPT side

**Review by flogross89 — CHANGES_REQUESTED** [review]
thanks!

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml (hunk 1 above)
Comment (tracernz) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml:29]: The FBW_Push_Held template already adds this code.

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml (hunk 2 above)
Comment (tracernz) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml:30]: We don't have this for the other buttons that are already hooked up and I don't have this kind of info for the A380.

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml (hunk 1 above)
Comment (flogross89) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/glareshield.xml:29]: roger

**Review by flogross89 — APPROVED** [review]

**Pull request merged**